packages/components/src/gradient-picker/index.js

```diff
@@ -1,3 +1,5 @@
+// @ts-nocheck
```

Comment:
Now that this is [added to tsconfig.json](https://github.com/WordPress/gutenberg/pull/47764/files#r1125721058), it has to have a `@ts-nocheck` to ignore its TS errors.